docs/_tutorials/persistence-with-queues.md

```diff
@@ -76,93 +76,56 @@ This tutorial is available in [GitHub]({{ site.repository }}){:target="_blank"}
 
 At the end, this tutorial walks through downloading and running the sample from source.
 
-## Configuring Solace JNDI
 
-In order for JMS clients to successfully connect to a message broker like a Solace message router, they need to look up a connection factory object using the Java Naming and Directory Interface (JNDI) service. Solace message routers provide a JNDI lookup service to make this integration easy.
 
-Additionally, this tutorial will make use of one JNDI topic for publishing and subscribing to messages. For simplicity in this tutorial, the following CLI script is provided. It will do the following:
+## Solace message router JMS administered objects
 
-*   Properly configure the default message-VPN with the necessary JNDI configuration.
-*   Create the required durable Queue
+This tutorial will make use of two JMS administered objects:
+*   A ConnectionFactory object – Used by JMS clients to successfully connect to a message broker like a Solace message router
+*   A Queue Destination – Used for publishing and subscribing to messages. This example will use the queue `Q/tutorial`
 
-The script configures these required JNDI resources:
+As described in the [publish/subscribe tutorial]({{ site.baseurl }}/publish-subscribe) we will use the approach of programmatically creating the required objects.
 
-<table>
-    <tr>
-        <th>Resource</th>
-        <th>Value</th>
-    </tr>
-    <tr>
-        <td>Connection Factory</td>
-        <td>/JNDI/CF/GettingStarted</td>
-    </tr>
-    <tr>
-        <td>Queue</td>
-        <td>/JNDI/Q/tutorial</td>
-    </tr>
-</table>
+A difference to the publish/subscribe tutorial is that here an additional physical endpoint resource – a durable queue, associated with the Queue Destination – is required on the message router, which will persist the messages until consumed.
 
-The script also creates a durable queue named “Q/tutorial” and enables this queue. This script assumes the connection factory and queue do not exist and creates them. If the JNDI connection factory or queue already exists you may need to remove the keyword “create” from the script below.
+Using the Solace Dynamic Durables property enables the physical endpoint to be dynamically created if it doesn’t exist yet, when the JMS Queue Destination is created programmatically. For other ways of creating Solace message router resources, refer to the [Solace Documentation - Management Tools]({{ site.docs-management_tools }}){:target="_top"}
 
```

Comment:
An alternative approach would be:

1) Here is an interesting feature X. It has the following properties.
2) This tutorial will make use of X and here is the code for how to do it.

That's also fine. IMO, the original text is half way between both approaches.